### PR TITLE
chore: remove paginator dependency and use gapic to return streams for paginated calls instead

### PR DESCRIPTION
**Pull request by AVaksman**
gapic supports both paginated calls and its streamified versions. 

- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by stephenplusplus** [comment]
@AVaksman I know it's still a draft, so sorry if I'm jumping in too soon. But, I was wondering if you could describe the intentions behind this when you get the chance? Thanks!

**Comment by AVaksman** [comment]
gapic now supports paginated calls and its streamified versions directly.

**Comment by stephenplusplus** [comment]
Does it also retry requests?

**Comment by AVaksman** [comment]
Yes, gax supports retries for paged calls.
`RetryOptions{retryCodes; backoffSettings;}` can be passed via `gaxOptions.retry` argument.
Streamified version of paged call is essentially same paged call where results pushed one by one via an object mode stream.

**Review by stephenplusplus — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -750,16 +750,20 @@ export class Bigtable {
       });
     };
 
+    const streamified = {listTablesStream: true};
```
[the remark this answers is not in the document]
Comment (AVaksman) [reply src/index.ts]: Done

File: src/index.ts [hunk 2]
```diff
@@ -805,6 +809,20 @@ export class Bigtable {
           .pipe(stream);
       });
     }
+
+    function makeRequestStreamified() {
```
[the remark this answers is not in the document]
Comment (AVaksman) [reply src/index.ts]: Done

File: src/instance.ts [hunk 3]
```diff
@@ -905,6 +908,71 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
...
+      parent: this.name,
+      view: Table.VIEWS[options.view || 'unspecified'],
+    });
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    delete (reqOpts as any).gaxOptions;
+
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
+    const self = this;
+    const transform = function (
```
[the remark this answers is not in the document]
Comment (AVaksman) [reply src/instance.ts]: Done!
Have to pass tableId only thought, Table constructor checks for the proper projectId presence.

**Review by stephenplusplus — APPROVED** [review]

**Comment by stephenplusplus** [comment]
Thanks!

**Comment by AVaksman** [comment]
@stephenplusplus I added another [unit test](https://github.com/googleapis/nodejs-bigtable/pull/762/commits/e9839fdfc8cfd9eaf12aaa343701215400c9af04) after the approval

**Comment by stephenplusplus** [comment]
Nice, lgtm!

**Pull request merged**